### ISSUE TITLE
Not Working IOS 14.3 Unc0ver

### ISSUE THREAD

**Issue by Funstab** [LemaMichael/Unicorn#21]
IOS 14.3 
unc0ver 6.0.2 
Iphone X 

Issues: 
 
No download button
Remove everything on screen does not work

(Unicorn shows up in settings and some features work but most of them don't)

After downgrading TikTok to the supported version (18.5) it still won't work.

**Comment by rastafaa** [comment]
It stopped working on 13.5 too.. think TikTok managed to block it

**Comment by sam901-8** [comment]
Yes same here.. i noticed that the download button that lets you downlpad videos have disappeared

**Comment by Funstab** [comment]
> Yes same here.. i noticed that the download button that lets you downlpad videos have disappeared

Open settings and make sure appstore doesn't automatically update apps, after that download the Appstore++ tweak and use that to downgrade to TikTok 18.1 that will make the tweak fully function again.

**Comment by ron8841** [comment]
same issue with sideloading the app, download button has disappeared, since tiktok has been sideloaded it hasn't updated app version

**Comment by codster2** [comment]
mine has some functions but ones like the spoof likes and view private videos/under review videos isn't there

**Comment by codesterd** [comment]
> same issue with sideloading the app, download button has disappeared, since tiktok has been sideloaded it hasn't updated app version

The sideloaded app on AppDB still works and has the download button. Just tried it last night and using it over the jailbreak tweak. It's version 4.9 of Unicorn and 18.5.0 of TikTok.

**Comment by ron8841** [comment]
> > same issue with sideloading the app, download button has disappeared, since tiktok has been sideloaded it hasn't updated app version
> 
> The sideloaded app on AppDB still works and has the download button. Just tried it last night and using it over the jailbreak tweak. It's version 4.9 of Unicorn and 18.5.0 of TikTok. 

just reinstalled it with this version and through appDB, still the same issues

**Comment by rastafaa** [comment]
Downgrading tiktok last week fixed the issue for me.

**Comment by ron8841** [comment]
> Downgrading tiktok last week fixed the issue for me.

older versions dont work with ios 14.5
